### DISCUSSION TITLE
Any plugin to autoclose unedited buffers

### DISCUSSION THREAD

**Discussion by ctretyak** [AstroNvim/astrocommunity#1282] [General]
Guys, Do you know any plugins that work like vscode tabs options "workbench.editor.limit" and don't allow you to open more than N tabs in one editor and autoclose the oldest ones?

**Comment by luxus** [comment]
not sure if this is the right place to ask, maybe try /r/neovim or our discord in the #neovim channel

**Reply by ctretyak** [reply]
https://www.reddit.com/r/neovim/comments/1h81p08/any_plugin_to_autoclose_unedited_buffers/

**Comment by ctretyak** [comment]
Some guy on Reddit suggested me https://github.com/axkirillov/hbac.nvim. This is solved the problem.
